examples/cra-auth/src/index.tsx

```diff
@@ -46,9 +46,11 @@ ReactDOM.render(
         },
       }}
     >
-      <AuthProvider translations={translations} initialLocale="pt-BR">
-        <App />
-      </AuthProvider>
+      <NotificationsProvider>
+        <AuthProvider translations={translations} initialLocale="pt-BR">
```

Comment:
I have a suggestion. I think it'll be better if we centralize the translations on `I18nProvider`.

Instead of doing this:

```tsx
<AuthProvider translations={translationsAuth} initialLocale="pt-BR">
  <MapsProvider translations={translationsMaps} initialLocale="pt-BR">
    <App translations={translationsApp} initialLocale="pt-BR" />
  </MapsProvider>
</AuthProvider>
```

We do this:

```tsx
const translations = // all translations;

<I18nProvider translations={translations} initialLocale="pt-BR">
  <AuthProvider>
    <MapsProvider>
      <App />
    </MapsProvider>
  </AuthProvider>
<I18nProvider>
```

This way, all packages, and the application will get the translations.

Reply:
https://github.com/ttoss/modules/issues/55